app/assets/stylesheets/local/_questions.scss

```diff
@@ -91,7 +91,8 @@ form.questioning_form {
   z-index: 500;
 
   .modal-body {
-    min-height: 400px;
+    height: 400px;
+    overflow: auto;
```

Comment:
@ziedAb Can you explain what happened here? Was the modal getting too tall? Is 400px a good size for the fixed height? Could we maybe go taller? Can you provide a screenshot with an option set with around 10 options on it?

Reply:
![400px modal height](https://user-images.githubusercontent.com/5773213/65324346-e2a30f00-dba3-11e9-8a79-1a8bf2e2cd79.png)

400px isn't the size of the whole modal, it is just the body

Reply:
Thank you.